### ISSUE TITLE
Display math in knowls contains LaTeX labels

### ISSUE THREAD

**Issue by rbeezer** [rbeezer/mathbook#532]
As duplicate content, this should not happen, and can manifest as un-rendered LaTeX/MathJax.

Does not seem to be new (ie due to recent display math changes).  For sample article, only instances reported by `grep` are

```
corollary-FTC-derivative.html:<p id="p-31">Suppose \(f(x)\) is a continuous function.  Then \begin{equation}\frac{d}{dx}\definiteintegral{a}{x}{f(t)}{t}=f(x)\text{.}\label{equation-alternate-FTC}\tag{4.1}\end{equation}</p></article><div class="posterior"><div><a knowl="./knowl/proof-FTC-corollary.html" knowl-id="xref-proof-FTC-corollary" alt="Proof 4.1.1 " title="Proof 4.1.1 ">Proof</a></div></div>
proof-FTC-corollary.html:\frac{d}{dx}\definiteintegral{a}{x}{f(t)}{t}&amp;=\frac{d}{dx}\left(F(x)-F(a)\right)\label{equation-use-FTC}\tag{4.2}\\
proof-FTC-corollary.html:&amp;=f(x)-0 = f(x)\label{mrow-3}\tag{4.3}
```

**Comment by Alex-Jordan** [comment]
At:
http://spot.pcc.edu/math/APEXCalculus/sec_differentials.html

There is a knowl to Equation (4.4.1). When it opens, the math content is not rendered. No error is reported; it just prints the MathJax input. However I think the issue is the same as this one, and that the strange behavior is connect to the repeated label. If I put this content into, say, math.se, it renders fine. But if I paste a second instance, the second one gives me an error that the label has already been used.

**Comment by mitchkeller** [comment]
The knowl that opens when you click the 7.14 in "Our proof of 7.14 requires" on http://rellek.net/book/s_inclusion-exclusion_euler-phi.html
also shows unrendered MathJax input. Also appears to be related to a repeated label. Thanks to @davidfarmer for noting this on my issue tracker. I'm going to close my issue there (having resolved in source the missing `@autoname` on some `xref`s in that vicinity) and await word on a fix over here.

**Comment by rplantz** [comment]
Math is not rendered in the knowl that opens when you click on "Equation 5.1.1" at [http://bob.cs.sonoma.edu/IntroCompOrg-RPi/sec-balgebra.html](url). (May have to scroll down a bit to see the knowl that opens.)

**Comment by rbeezer** [comment]
Is it a duplicate \label?  If so, there is an issue you can add this to.  I made progress last weekend on a more general fix for duplicated content, which is proving tricky.

**Comment by rplantz** [comment]
Not sure how to look for duplicate \label. But when I change:

   <p><ul>
      <li><p>AND and OR are <term>associative</term>:<index><main>Boolean
algebra properties</main><sub>associative</sub></index><mdn>
            <mrow xml:id="and-ass">x \cdot (y \cdot z) = (x \cdot y) \cdot
z</mrow>
            <mrow xml:id="or-ass">x + (y + z) = (x + y) + z</mrow>
         </mdn></p>
         <p>For <xref ref="and-ass">Equation</xref>:</p>

to:
   <p><ul>
      <li><p>AND and OR are <term>associative</term>:<index><main>Boolean
algebra properties</main><sub>associative</sub></index><men
xml:id="and-ass">x \cdot (y \cdot z) = (x \cdot y) \cdot z</men>
            <men xml:id="or-ass">x + (y + z) = (x + y) + z</men></p>
         <p>For <xref ref="and-ass">Equation</xref>:</p>

the knowl renders very nicely. Seems like the problem I'm having is with
<mdn>. Should I be using <men> in this application?

--Bob

**Comment by rbeezer** [comment]
The men will make a numbed equation which will have a \label which will mistakenly migrate to the knowl which when opened will be a duplicate of the original that is on the same page which will confuse MathJax and then not render any of the math.

Fix is obvious but systemic and is in-progress.

Please find the issue and link back to this discussion and you can check back after the fix?

Thanks,
Rob

**Comment by rbeezer** [comment]
This is similar to #222, which is the LaTeX version of the same problem.   As before:  "Fix is obvious but systemic and is in-progress."

**Comment by rbeezer** [comment]
Also, #176 was the previous issue I'd been thinking of.

This should be fixed as of 72df757a.

Once you rebuild, could you check reports made on this thread?
